cluster/gce/gci/configure-helper.sh

```diff
@@ -156,9 +156,14 @@ function main() {
   reset-motd
   prepare-mounter-rootfs
   modprobe configs
-  if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    start-prometheus &  #####start prometheus
+  if [[ "${ENABLE_PPROF_DEBUG:-false}" == "true" ]]; then
+  start-collect-pprof &  #### start collect profiling files
   fi
+  if [[ "${ENABLE_PROMETHEUS_DEBUG:-false}" == "true" ]]; then
+    start-prometheus &  #####start prometheus
+  fi 
+  ulimit -c unlimited
+
   echo "Done for the configuration for kubernetes"
```

Comment:
@vinaykul, I changed this section, this prometheus is for debug purpose, it is not only for scale out, so I added an dedicated env var. Please let me know if you have any concern.

Reply:
It looks fine to me, but let's get @pdgetrf buy-off.